ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -2564,62 +2577,64 @@
     },
     "svCM": {
         "long_name": "Sound Velocity",
-        "units": "m/s [Chen-Millero]",
+        "units": "m/s",
+        "bodc_alternative_label": "CTD_Sound_Vel",
         "standard_name": "speed_of_sound_in_sea_water",
         "sdn_parameter_urn": "SDN:P01::SVELCA01",
-        "sdn_parameter_name": "Sound velocity in the water body by computation from salinity and temperature using UNESCO algorithm",
+        "sdn_parameter_name": "Sound velocity in the water body by computation from salinity and temperature using UNESCO algorithm [Chen-Millero]",
         "sdn_uom_urn": "SDN:P06::UVAA",
         "sdn_uom_name": "Metres per second",
         "comments": "1st sensor"
     },
     "svCF": {
         "long_name": "Sound Velocity",
-        "units": "ft/s [Chen-Millero]",
+        "units": "ft/s",
```

Comment:
like above

Reply:
I removed it in order to have just units inside the "units" parameter. If we leave something else than the unit there, some application may just crash or create some ambiguous output when retrieving the units. We may look for some parameter name to be added here to keep just the algorithm name used for the variable retrieval from raw measurements.

Reply:
yeah right good point, CF recommend to be compatible with UDUNITS. It makes sense to have this elsewhere. Though as along as it is represented elsehere